src/auro-radio.scss

```diff
@@ -106,23 +109,28 @@
 .label {
   display: block;
 
-  margin-left: var(--auro-size-xl);
+  margin-left: calc(var(--auro-size-300));
+
+  &:hover:after {
+    background-color: var(--auro-color-ui-bkg-hover-on-light);
+  }
+
+  &:focus-visible:after {
+    border-color: var(--auro-color-border-primary-on-light);
+  }
 
   &:after {
     position: absolute;
     z-index: 1;
-    top: var(--auro-size-xxs);
-    left: var(--auro-size-xs);
+    top: var(--auro-size-50);
+    left: var(--auro-size-50);
 
-    width: var(--auro-size-lg);
-    height: var(--auro-size-lg);
+    width: calc(var(--auro-size-200) + var(--auro-size-50));
+    height: calc(var(--auro-size-200) + var(--auro-size-50));
 
     content: '';
 
     border-radius: 50%;
-  }
-
-  &:after {
```

Comment:
The same change is on the other PR? Why was this removed?

Reply:
It was redundant.